src/cryptonote_protocol/levin_notify.cpp

```diff
@@ -98,6 +98,16 @@ namespace levin
       return std::chrono::steady_clock::duration{crypto::rand_range(rep(0), range.count())};
     }
 
+    bool set_connection_can_broadcast_txs_to_peer(std::shared_ptr<connections> p2p, boost::uuids::uuid conn, bool val, bool exclusive_peers) {
+      if (exclusive_peers && !val)
```

Comment:
Why even have `val`? It looks to always be `false` ? Basically this function is designed to disable sending, and never re-enable it.

Reply:
This was meant to be temporary, sorry you had to see it.